helix-rest/src/main/java/org/apache/helix/rest/metadatastore/RoutingData.java

```diff
@@ -0,0 +1,45 @@
+package org.apache.helix.rest.metadatastore;
+
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import java.util.Map;
+
+public interface RoutingData {
```

Comment:
Shall we just call it ZkRealmData?
It might be confusing because we have routing table provider for completely different usage.

Reply:
I don't have the context for the routing table provider, therefore I don't have any objection to that. Tagging @narendly for reference.

Reply:
As we discussed, "MetadataStoreRoutingData" might be more appropriate here.